CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## [1.58.125](https://github.com/brave/brave-browser/releases/tag/v1.58.125)
+
+### Web3
+
+ - Added Filecoin to Brave Wallet. ([#30342](https://github.com/brave/brave-browser/issues/30342))
+ - Added support for EIP-6963. ([#30595](https://github.com/brave/brave-browser/issues/30595))
+ - Added Solana to the top of the network list. ([#31506](https://github.com/brave/brave-browser/issues/31506))
+ - Added network selector in the "Add custom asset" dialog. ([#31188](https://github.com/brave/brave-browser/issues/31188))
+ - Added network selector in the "Edit visible assets" dialog. ([#27093](https://github.com/brave/brave-browser/issues/27093))
+ - Implemented web UI for "Send" similar to desktop. ([#31260](https://github.com/brave/brave-browser/issues/31260))
+ - Implemented web UI for "Buy" similar to desktop. ([#31494](https://github.com/brave/brave-browser/issues/31494))
+ - Implemented web UI for "Deposit" similar to desktop. ([#31495](https://github.com/brave/brave-browser/issues/31495))
+ - [Security] Fixed Brave Wallet pop-up hiding the full screen toast as reported on HackerOne by shadow2639. ([#30113](https://github.com/brave/brave-browser/issues/30113))
+ - Improved the flow for manually adding NFTs. ([#31399](https://github.com/brave/brave-browser/issues/31399))
+ - Removed reference to Brave Rewards within the wallet reset flow. ([#28003](https://github.com/brave/brave-browser/issues/28003))
+ - Updated "Add custom asset" dialog to appear in full screen (also added "Add NFT" screen which appears in full screen). ([#27070](https://github.com/brave/brave-browser/issues/27070))
+ - Fixed certain wallet UI components still being visible after wallet has been locked. ([#31272](https://github.com/brave/brave-browser/issues/31272)), ([#23049](https://github.com/brave/brave-browser/issues/23049)), ([#31236](https://github.com/brave/brave-browser/issues/31236))
```

Comment:
Maybe update to `Fixed transaction confirmation screen being shown when wallet gets auto-locked.`

Reply:
With the above, there was a few of these re: UI elements still being visible. Rather than doing three lines of basically the same issue, I bundled them into a single one.

Reply:
#31236 is for the webui change. 31272 was specifically for wallet screen hence updated the note.

Reply:
Done.